tests/test_coastal_vulnerability.py

```diff
@@ -1167,6 +1167,57 @@ def test_aoi_multiple_features(self):
         n_points = layer.GetFeatureCount()
         self.assertTrue(n_points == 6)
 
+    def test_landmass_interior_holes(self):
+        """CV: test no shore points created on interior holes."""
+        workspace_dir = self.workspace_dir
+
+        aoi_path = os.path.join(workspace_dir, 'aoi.geojson')
+        landmass_path = os.path.join(workspace_dir, 'landmass.geojson')
+        polygon_pickle = os.path.join(
+            workspace_dir, 'polygon.pickle')
+        lines_pickle = os.path.join(
+            workspace_dir, 'lines.pickle')
+        lines_rtree = os.path.join(
+            workspace_dir, 'rtree.dat')
+        target_vector_path = os.path.join(
+            workspace_dir, 'shore_points.gpkg')
+        model_resolution = 100
+        srs = osr.SpatialReference()
+        srs.ImportFromEPSG(26910)  # UTM Zone 10N
+        wkt = srs.ExportToWkt()
+
+        aoi_poly = Polygon([
+            (-200, -200), (200, -200), (200, 200), (-200, 200), (-200, -200)])

```

Comment:
Since we're defining a box, I wonder if [`shapely.geometry.box`](https://shapely.readthedocs.io/en/stable/manual.html#shapely.geometry.box) might be slightly easier for future us to grok?

Reply:
Actually, please disregard ... the use of coordinates here is consistent with the very sensible use of coordinates later in the test.